test/e2e/rule_test.go

```diff
@@ -515,10 +515,10 @@ func TestRule_CanRemoteWriteData(t *testing.T) {
 				Scheme: "http",
 			},
 		},
-	}, &config.RemoteWriteConfig{
+	}, []*config.RemoteWriteConfig{{
 		URL:  &common_cfg.URL{URL: rwURL},
 		Name: "thanos-receiver",
-	})
+	}})
```

Comment:
Worth a test case for multiple remote write targets?

Reply:
Hm. maybe not, since it's the Agent logic? We don't want to test Agent really

Reply:
whatever, let's do it ;p

Reply:
done